functions/compose.php

```diff
@@ -18,7 +18,7 @@
  *
  * Takes n function arguments.
  *
- * @param  callable[] ...$args
+ * @param  callable|callable-string ...$args
```

Comment:
How does this work with method calls? I'm assuming `callable-string` cannot contain `[$myObject, "myMethod"]`.

Reply:
I've added a unit test and PHPStan checks those classes. It doesn't complain when giving `compose` a callable like that. I think PHP / PHPStan knows the difference between a random array and a callable.